File: LeetCode-Problems/Easy/Palindrome-Number.java
//Java
class Solution {
    public boolean isPalindrome(int x) {
        if (x < 0){
            return false;
        }
        if (x > 0 && x < 10){
            return true;
        }
        String palindrome = Integer.toString(x);
        String reverse = "";
        for (int i = palindrome.length()-1; i >= 0; i--){
            reverse += palindrome.substring(i, i+1);
        }
        if (reverse.equals(palindrome)){
            return true;
        }
        else{
            return false;
        }
    }
}

File: LeetCode-Problems/Easy/Palindrome-Number.py
# Python
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        y = str(x)
        palindrome = ""
        if (x < 0):
            return False
        if (x > 0 and x < 10):
            return True
        for x in range (len(y), -1, -1):
            palindrome += y[x:x+1]
        if (palindrome == y):
            return True
        return False
        
        
